tests/artifactorylocalrepository_test.go

```diff
@@ -46,9 +47,126 @@ func localMavenTest(t *testing.T) {
 	if err != nil {
 		t.Error("Failed to update " + repoKey)
 	}
```

Comment:
I suggest replacing all if err != nil to assert.NoError:
```suggestion
      assert.NoError(t, err, "Failed to update " + repoKey)
```